test/agentchat/contrib/test_qdrant_retrievechat.py

```diff
@@ -3,6 +3,7 @@
 import pytest
 from autogen.agentchat.contrib.retrieve_assistant_agent import RetrieveAssistantAgent
 from autogen import config_list_from_json
+from conftest import skip_openai
```

Comment:
The import will fail if we directly run this python file.

Reply:
Would the fix from #1144 work here as well?

```
sys.path.append(os.path.join(os.path.dirname(__file__), "../.."))
from conftest import skip_openai  # noqa: E402

```